src/projectApplications.js

```diff
@@ -0,0 +1,137 @@
+const fs = require('fs');
+const path = require('path');
+const {google} = require('googleapis');
+
+const SCOPES = ['https://www.googleapis.com/auth/spreadsheets.readonly'];
+const ALL_QUARTERS_ID = '16vmuNiN-8mij8EoEIwvQ_wde9DgF_0IWXfT_95SHfcc';
+
+const numOfQuarters = 14;
+
+// Load client secrets from a service account
+fs.readFile(path.resolve(__dirname, '../credentials.json'), (err, content) => {
+  authorize(JSON.parse(content), main);
+});
+
+function authorize(credentials, callback) {
+  const jwt = new google.auth.JWT(credentials.client_email, null, credentials.private_key, SCOPES);
+  callback(jwt);
+}
+
+function main(auth) {
+  process.stdout.write('Parsing project sheet .. ');
+  readProjectMemberSheet(auth);
+}
+
+function readProjectMemberSheet(auth) {
+  const sheets = google.sheets({version: 'v4', auth});
+  const endQuarter = ['Winter', '2021'];
+  const startQuarter = ['Fall', '2016'];
+  const quarters = ['Winter', 'Spring', 'Fall'];
+  let current = startQuarter;
+  let masterDict = [];
+  let count = 0;
+
+  
+  while(current[1] < endQuarter[1] ||
+    (current[1] == endQuarter[1] && quarters.indexOf(current[0]) <= quarters.indexOf(endQuarter[0]))){
+    
+    const currentQuarter = current[0];
+    const currentYear = current[1];
+    // Run on current quarter
+    sheets.spreadsheets.values.get({
+      spreadsheetId: ALL_QUARTERS_ID,
+      range: `${current[0]} ${current[1]}!A:Z`,
+    }).then((response) => {
+      const info = formatProjectMemberResponse(response);
+      // console.log(info);
+      let memberinfo = [];
+      let quarter = [];
+      for(var i = 0; i<info.length; i++) {
```

Comment:
your issue is that you are using `var` instead of `let` here.

read [this stackoverflow](https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var#:~:text=The%20main%20difference%20is%20the,outside%20the%20loop%20for%20example.&text=let%20allows%20you%20to%20declare,on%20which%20it%20is%20used.) post to learn more about it